versions.tf

```diff
@@ -3,13 +3,15 @@ terraform {
   required_version = ">= 0.13"
```

Comment:
Did you mean for this to be 0.14 too?

Reply:
No, this is OK to remain at 0.13 since it's a module and doesn't have a lock file.